classes/02class/exercises/c02-network01/mendoncaj/README.md

```diff
@@ -0,0 +1,47 @@
+# c02-network01
+
+## Commands Execution Output
+
+- Answers for `10.0.0.1/24`:
+
+  - IP in binary: `00001010.00000000.00000000.00000001`
+  - subnet mask in binary: `11111111.11111111.11111111.00000000` 
+  - subnet mask in decimals: `255.255.255.0`
+  - private/public: `private` ( Class A )
+  - hosts: `251` ( considering AWS reserved ips )
+
+- Answers for `192.168.0.1/16`:
+
+  - IP in binary: `11000000.10101000.00000000.00000001`
+  - subnet mask in binary: `11111111.11111111.00000000.00000000` 
+  - subnet mask in decimals: `255.255.0.0`
+  - private/public: `private` ( Class C )
+  - hosts: `65.531` ( considering AWS reserved ips )
+
+- Answers for `249.165.166.135/30`:
+
+  - IP in binary: `11111001.10100101.10100110.10000111`
+  - subnet mask in binary: `11111111.11111111.11111111.11111100` 
+  - subnet mask in decimals: `255.255.255.252`
+  - private/public: `public` ( Class E - Experimental )
+  - hosts: `NA` ( considering AWS reserved ips )
```

Comment:
If not talking AWS, this is used in P2P connections (network with only two hosts) and is allowed to exist.